libs/sdk/src/NativeBridge.ts

```diff
@@ -14,6 +15,7 @@ export interface Configuration {
   autoSessionTracking?: boolean;
   sessionTimeout?: number;
   enableBackgroundMode?: boolean;
+  dbEncryption?: RNDBEncryption | null;
```

Comment:
Instead of making `dbEncryption` to accept `null`, and then checking if it is non null in `configure()`, we can check for undefined right ?